src/core/public/application/application_service.tsx

```diff
@@ -320,7 +320,7 @@ export class ApplicationService {
       navigateToApp,
       navigateToUrl: async (
         url: string,
-        { skipAppLeave = false, forceRedirect = false }: NavigateToUrlOptions = {}
+        { skipAppLeave = false, forceRedirect = false, state }: NavigateToUrlOptions = {}
```

Comment:
Please add the associated unit test here: https://github.com/elastic/kibana/blob/3c3aa3354aa2717bcdf91effb9d8a1181af779bd/src/core/public/application/application_service.test.ts#L1098

Reply:
I just added it